scripts/umbrel-os/services/umbrel-external-storage.service

```
@@ -8,6 +8,7 @@ Description=External Storage Mounter
 Type=oneshot
 Restart=no
 ExecStart=/home/umbrel/umbrel/scripts/umbrel-os/external-storage/mount
+ExecStartPost=/home/umbrel/umbrel/scripts/umbrel-os/external-storage/update-from-sdcard
```

Do you think it's a good idea to increase the default timeout time from the default 90 seconds to something that's much more generous, like 45 minutes? The reason being that if there's a version mismatch and `update-from-sdcard` triggers the OTA update process, it could take a long while to download and extract new Docker images, especially on a slow network.

```suggestion
ExecStartPost=/home/umbrel/umbrel/scripts/umbrel-os/external-storage/update-from-sdcard
TimeoutStartSec=45min
```